example_configs/tomcat.json

```diff
@@ -0,0 +1,44 @@
+{
+  "rules": [
+    {
+      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>(\w+):",
+      "name": "tomcat_$3_total",
+      "labels": {
+        "protocol":"$1",
+        "port":"$2" },
+      "help": "Tomcat global $3",
+      "type": "COUNTER",
```

Comment:
All attributes of GlobalRequestProcessor can only increase. The only decrease after reboot the system, overflow or you manually reset the counters.